main.py

```diff
@@ -34,6 +36,10 @@
         backend=default_backend()
     )
 
+# Import other pubkeys
```

Comment:
Relative hard coded paths 🤔

Reply:
Relative paths rely on the current directory of the program, this can change depending on the execution of the scirpt. One way to get around this is to use `__file__` which gets the location of the file. You can use this to determine where the path of the file is and get to the desired location. 

This will give you the directory name of the file, which you can use to append to the path of desired location, this case the keys.
```
os.path.dirname(__file__)
```

If you want to go further you can create a config file and import that pubkey path from the file, this allows the keys to be placed in a completely user defined location. Whatever works best for you.

Reply:
Hopefully fixed in [fea9271](https://github.com/RobinsMediaTeam/Cardboard/pull/21/commits/fea9271246d54c52bf1c93151ae74b8c901cabae)